migrations/20190207220000_init_db.js

```diff
@@ -0,0 +1,355 @@
+const initialize = async (knex, Promise) => {
```

Comment:
Is this file an initial migration that folks need to run to get everything set up on the new system? It seems like the knex CLI should be covering us for future migrations. If it is an initial migration that needs to be run, can we label it with a comment block and document it?

Reply:
it is an initial migration, but does not need to be run manually, because of this:
https://github.com/MoveOnOrg/Spoke/pull/1154/files#diff-6e0d62f54b853b53af874f5965d7adf2R36
i.e. on startup, by default migrations or initial db creation happen automatically.
So knex CLI is mostly useful for creating new migrations or manipulating them for db admin or doing it manually, e.g. in a production setting, where you want to run manually instead of let it 'autorun'